CONTRIBUTING.md

```diff
@@ -121,9 +121,11 @@ This will publish to the Gradle Portal and upload to OSSRH. After a successful C
 to https://oss.sonatype.org/ and release the artifacts manually. This step is called "close, release and drop" in the
 Sonatype ecosystem.
 
+> 📣 Before publicizing the release, wait for it to be visible on [Maven Central](https://repo1.maven.org/maven2/com/apollographql/apollo3/) (this usually takes a few minutes).
+
```

Comment:
Put this in the bullet points so we can read things serially? My brain is super bad at multi-tasking 😅 

* `scripts/tag.main.kts <version-name>`
* push the tag and branch
* while it compiles, prepare the changelog, open a PR to `CHANGELOG.md` 
* wait for the CI to finish compiling
* go to https://oss.sonatype.org/, and release the artifacts manually. This step is called "close, release and drop" in the Sonatype ecosystem.
* wait for the release to be online
* paste the changelog in a new release on [GitHub](https://github.com/apollographql/apollo-android/releases)
* merge pending documentation/tutorial updates
* if it's a significant release, tweet about it 🐦
* relax 🍹